src/pombot/index.js

```diff
@@ -0,0 +1,30 @@
+import {RtmClient, WebClient, MemoryDataStore} from '@slack/client';
+import {createSlackBot, createConversation, createCommand} from 'chatter';
+import config from '../../config';
+
+// define commands
+import helpCommand from './commands/help';
+
+// create bot
+const bot = createSlackBot({
+  name: 'Pombot',
+  slack: {
+    rtmClient: new RtmClient(config.tokens.pombot, {
+      dataStore: new MemoryDataStore(),
+      autoReconnect: true,
+    }),
+    webClient: new WebClient(config.tokens.pombot),
+  },
+  createMessageHandler() {
+    return createConversation([
+      this.createSlackMessageHandler({dm: true}, [
+        helpCommand
```

Comment:
DM command:
https://github.com/bocoup/bot-framework/blob/483cdd6b54b5f8d9b70399c0535fde530d5c4da5/examples/slack.js#L126-L136

Public command:
https://github.com/bocoup/bot-framework/blob/483cdd6b54b5f8d9b70399c0535fde530d5c4da5/examples/slack.js#L141-L149

Notice they're the same, except the public one has a name.